src/Connectivity/Connectivity.jl

```diff
@@ -1,6 +1,9 @@
 module Connectivity
 
 using LightGraphs
+using LightGraphs: is_graphical
+using LightGraphs.SimpleGraphsCore: SimpleDiGraph
+
```

Comment:
In many cases I took the opportunity to be specific about what we're using. In this case, like Community, a function (condensation) requires the ability to create a graph structure. So it's using SimpleGraphsCore.